docs/guide/tensorboard.rst

```diff
@@ -80,3 +80,44 @@ Here is a simple example on how to log both additional tensor or arbitrary scala
 
 
     model.learn(50000, callback=TensorboardCallback())
+
+Logging Videos
+--------------
+
+Tensorboard supports periodic logging of video data, which helps evaluating agents at various stages during training.
+Here is an example of how to render an episode and log the resulting video to tensorboard at regular intervals:
+
+.. code-block:: python
+
+    import gym
+    import torch as th
+
+    from stable_baselines3 import A2C
+    from stable_baselines3.common.callbacks import BaseCallback
+    from stable_baselines3.common.evaluation import evaluate_policy
+    from stable_baselines3.common.logger import Video
+    from typing import Any, Dict
+
+
+    class ReportTrajectoryCallback(BaseCallback):
```

Comment:
maybe `VideoRecorderCallback` would be more appropriate?

Reply:
Yes!